### DISCUSSION TITLE
Adding your examples to my Free Pascal study notes, aka. Cookbook

### DISCUSSION THREAD

**Discussion by ikelaiah** [mr-highball/ezthreads#20] [General]
Hi,

Many thanks for this library. I'm learning it right now as I find the learning curve of TThread is quite steep.

Would you mind if I add your example in my Free Pascal study note?
URL:  https://ikelaiah.github.io/free-pascal-and-lazarus-cookbook/

Also, do let me know if you are accepting financial contributions. I have PayPal.

Many thanks,
Iwan

**Comment by mr-highball** [comment]
Thanks @ikelaiah 🍻

By all means add whatever you'd like there. As for contributions, don't worry about it, it just makes me happy people find use with my code 

Cheers,
-Highball

**Reply by ikelaiah** [reply]
Thank you. I will add your examples to my cookbook. Hopefully, all this info will help others to ease into Free Pascal and Multi Threading easily.

Regards,
Iwan

**Reply by ikelaiah** [reply]
Hi @mr-highball ,

I've added three of your examples in my cookbook here; https://ikelaiah.github.io/free-pascal-and-lazarus-cookbook/docs/advanced/ezthreads/

I added more comments in them for my learning purpose.

All in all, ezthreads is pleasant to use.

Thanks,
Iwan

**Reply by mr-highball** [reply]
Happy to have helped :)
